KerbalVR_Mod/KerbalVR/InteractionSystem/KerbalVR_Hand.cs

```diff
@@ -168,53 +180,87 @@ public void Initialize(SteamVR_Input_Sources handType, Hand otherHand, bool isIV
 			SteamVR_Action_Boolean actionGrab = SteamVR_Input.GetBooleanAction("default", "GrabGrip");
 			actionGrab[handType].onChange += OnChangeGrab;
 
-			// set up skeleton helper
-			if (profile.useSkeletonHelper)
-			{
-				KerbalSkeletonHelper kerbalSkeletonHelper = handObject.AddComponent<KerbalSkeletonHelper>();
-				kerbalSkeletonHelper.retargetableSetting = profile.retargetableSetting;
-				kerbalSkeletonHelper.sourceSkeletonRoot = handSkeleton.skeletonRoot;
-				kerbalSkeletonHelper.destinationSkeletonRoot = handObject.transform.Find(profile.retargetableSetting.destinationSkeletonRootPath);
-			}
+			#endregion
+
+			#region Setup hand model
+
+			IVAProfile = handSetting.GetProfile(true);
+			SetupModel(IVAProfile, ref IVAObject);
+
+			EVAProfile = handSetting.GetProfile(false);
+			SetupModel(EVAProfile, ref EVAObject);
+
+			#endregion
+
+			#region Setup other stuffs
 
 			// set up ladder
 			ladder = gameObject.AddComponent<VRLadder>();
 
 			// set up UI hand  
 			UIHand = gameObject.AddComponent<VRUIHand>();
 
-			#region Setup Colliders
+			#endregion
+
+			#region Setup Collider
 
 			// add fingertip collider for "mouse clicks"
-			fingertipTransform = handObject.transform.Find(profile.indexTipTransformPath);
-			fingertipTransform.localPosition += profile.fingertipOffset;
+			fingertipTransform = new GameObject("KVR_FingertipCollider").transform;
+			fingertipTransform.SetParent(handObject.transform);
```

Comment:
Why do we attach this to the handobject and not the actual fingertip transform?  It seems like a waste of work to manually keep it updated all the time.

Reply:
Fixed by [c35e6bc](https://github.com/JonnyOThan/Kerbal-VR/pull/172/commits/c35e6bc15153e8ec6391f1a7d6308ce51420088a)